src/main/java/revai/RevAiWebSocketListener.java

```diff
@@ -0,0 +1,50 @@
+package revai;
+
+import okhttp3.Response;
+import revai.models.streaming.Hypothesis;
+
+/**
+ * Listens for events over the WebSocket connection to Rev.ai
+ */
+public interface RevAiWebSocketListener {
+
+  /**
+   * Supplies the connection message received from Rev.ai.
+   *
+   * @param message a String in JSON format that contains the message type and job ID.
```

Comment:
would make sense to parse this into a java class as well, eg it would be `onConnected(ConnectedMessage message)` where `ConnectedMessage` would have id/type properties (ConnectedMessage and Hypothesis could inherit from `StreamingResponseMessage base class which would have the type property)